### ISSUE TITLE
Add className to 'content' element

### ISSUE THREAD

**Issue by liubko** [quickleft/react-loader#13]
It might be useful to add className to 
```
return ( <div key="content" className="loadedContent">{this.props.children}</div> );
```

**Comment by levitanong** [comment]
+1. Also might be useful to take a clue from ReactTransitionGroup, wherein you can also choose the type of element, instead of just `div`s.

**Comment by mhuggins** [comment]
This has been addressed in #14, so I'll get another release out ASAP.  I'd like to add something to the README to call out the class name for reference first though.  I also am open to @levitanong's suggestion about passing in a component name, continuing to use a "div" as the default.  I'll open that as a separate enhancement request.

**Comment by mhuggins** [comment]
This change is now published under v1.2.0.
